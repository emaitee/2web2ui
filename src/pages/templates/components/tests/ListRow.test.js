import { shallow } from 'enzyme';
import React from 'react';

import ListRow from '../ListRow';

describe('ListRow', () => {
  let wrapper;
  Date = jest.fn((a) => a); // eslint-disable-line

  beforeEach(() => {
    const props = {
      published: true,
      id: 'id',
      name: 'name',
      last_update_time: 'date'
    };

    wrapper = shallow(<ListRow {...props} />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  it('should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
