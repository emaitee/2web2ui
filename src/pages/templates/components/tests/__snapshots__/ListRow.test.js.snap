// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListRow should render 1`] = `
<Table.Row
  key="id"
  rowData={
    Array [
      <Link
        replace={false}
        to="/templates/edit/id"
      >
        name
      </Link>,
      "id",
      "published",
      "date",
    ]
  }
/>
`;
